CHANGELOG.md

```diff
@@ -101,6 +113,34 @@ to avoid letting the user edit the immutable one.
   remainder of the cursor row after drawing rather than clearing the entire row
   before drawing, eliminating the "flicker" effect seen on some terminals.
 
+### Contributors
+
+Thanks to the people who made this release happen!
+
+* Alexander Potashev (@aspotashev)
```

Comment:
(Optional) I think we used to sort this by username? If so, it would be:

```
* Waleed Khan (@arxanas)
* Alexander Potashev (@aspotashev)
* Benjamin Tan (@bnjmnt4n)
* Glen Choo (@chooglen)
* Charles Crete (@Cretezy)
* Gregory Anders (@gpanders)
* Thomas Castiglione (@gulbanana)
* Danny Hooper (@hooper)
* Eidolon (@HybridEidolon)
* Ilya Grigoriev (@ilyagr)
* jyn (@jyn514)
* Théo Daron (@Kaporos)
* Matthew Davidson (@KingMob)
* Martin von Zweigbergk (@martinvonz)
* Matt Stark (@matts1)
* Michael Gattozzi (@mgattozzi)
* mlcui (@mlcui-google)
* Philip Metzger (@PhilipMetzger)
* Remo Senekowitsch (@senekor)
* Austin Seipp (@thoughtpolice)
* tinger (@tingerrr)
* Daniel Ploch (@torquestomp)
* Yuya Nishihara (@yuja)
```

Reply:
No, I guess we usually sort it like this:

```
* Alexander Potashev (@aspotashev)
* Austin Seipp (@thoughtpolice)
* Benjamin Tan (@bnjmnt4n)
* Charles Crete (@Cretezy)
* Daniel Ploch (@torquestomp)
* Danny Hooper (@hooper)
* Eidolon (@HybridEidolon)
* Glen Choo (@chooglen)
* Gregory Anders (@gpanders)
* Ilya Grigoriev (@ilyagr)
* jyn (@jyn514)
* Martin von Zweigbergk (@martinvonz)
* Matt Stark (@matts1)
* Matthew Davidson (@KingMob)
* Michael Gattozzi (@mgattozzi)
* mlcui (@mlcui-google)
* Philip Metzger (@PhilipMetzger)
* Remo Senekowitsch (@senekor)
* Thomas Castiglione (@gulbanana)
* Théo Daron (@Kaporos)
* tinger (@tingerrr)
* Waleed Khan (@arxanas)
* Yuya Nishihara (@yuja)
```

Reply:
I have sorted them by display name so far

Reply:
Yes, I just corrected it. I think you often capitalized the usernames too, but I just sorted it case-insensitively.

Reply:
I haven't capitalized usernames on purpose. I have tried to use exactly what the user's github page says, such as "
"Ilya Grigoriev" and "ilyagr" from https://github.com/ilyagr.

Reply:
That sounds fine to me. It's not a big deal in any case, feel free to merge this unsorted as far as I'm concerned.

Reply:
Yeah, I just ran `git log --format="%an" v0.17.1..HEAD | sort -u | awk '{print "* " $0}'` in order to get a sorted list of names, FWIW.

Reply:
Maybe it sorted by email address? I'm not sure how you ended up with

```
* Yuya Nishihara (@yuja)
* Daniel Ploch (@torquestomp)
```

in this order.

Reply:
Oh, you're right. I didn't even realize that. Well, `%an` is just printing the actual `user.name` from your Git config, no email is involved. Maybe `sort -u` is the wrong invocation for some reason? Of course, we should actually just automate all this, but that's a bit of a different discussion.